cdbot/cogs/fun.py

```diff
@@ -15,7 +15,7 @@
 from aiohttp import ClientSession
 from cdbot.constants import (
     ADMIN_ROLES, EMOJI_LETTERS, FAKE_ROLE_ID, QUOTES_BOT_ID, QUOTES_CHANNEL_ID,
-    STAFF_ROLE_ID, WELCOME_BOT_ID
+    STAFF_ROLE_ID, WELCOME_BOT_ID, Postgresql
```

Comment:
Little nitpick but can we call this `PostgreSQL`, looks nicer than Postgresql imo.